Faizan_Akhtar/project1-tic-tac-toe/assets/scripts/tic-tac-toe.js

```diff
@@ -0,0 +1,162 @@
+var gameSymbol = "X";
+
+function clicker(button){
+    if(gameSymbol == "X"){
+        gameSymbol = "O";
+        setButton(button, gameSymbol);
+    }
+    else if(gameSymbol == "O"){
+        gameSymbol = "X";
+        setButton(button, gameSymbol);
+    }
+}
+
+function btnDisable(){
+    document.getElementById("btn1").disabled = true;
+    document.getElementById("btn2").disabled = true;
+    document.getElementById("btn3").disabled = true;
+    document.getElementById("btn4").disabled = true;
+    document.getElementById("btn5").disabled = true;
+    document.getElementById("btn6").disabled = true;
+    document.getElementById("btn7").disabled = true;
+    document.getElementById("btn8").disabled = true;
+    document.getElementById("btn9").disabled = true;
+}
+
+function setButton(button, gameSymbol){
+    if(button == 1){
+        var btn = document.getElementById("btn1");
+        btn.innerHTML = gameSymbol;
+        btn.disabled = true;
+    }
+    else if(button == 2){
```

Comment:
Replace if-else ladder with switch to improve performance

Reply:
improved the code by adding a for loop